streamwebs_frontend/streamwebs/templates/streamwebs/site_detail.html

```diff
@@ -17,12 +17,12 @@
     var sheet_data = {{ data|safe }};
     var page_count = {{ pages }};
     var imageWidth = '200';
-  </script>
+</script>
+<!-- changed google maps key to AIzaSyBv7HB...-->
+<script src="https://maps.googleapis.com/maps/api/js?key=AIz
```

Comment:
This key needs to come from settings, and should not be hard-coded. Please fix to point to the configured key.